app/backend/Extensions/WebApplicationExtensions.cs

```diff
@@ -23,9 +23,19 @@ internal static WebApplication MapApi(this WebApplication app)
         // Get DALL-E image result from prompt
         api.MapPost("images", OnPostImagePromptAsync);
 
+        api.MapGet("enableLogout", OnGetEnableLogout);
+
         return app;
     }
 
+    private static IResult OnGetEnableLogout(HttpContext context)
+    {
+        var header = context.Request.Headers["X-MS-CLIENT-PRINCIPAL-ID"];
```

Comment:
If this HTTP header is available on server, wouldn't that mean that the client is sending it? If so, couldn't this be a client-side activity instead? The client would ask for an `HttpClient` and ask it for its current HTTP headers, no?

Reply:
That's a good question,,The redirecting api thing is just I don't know how to get `HttpContext` from blazor webassembly app.